twilio/twilio_connector.bal

```diff
@@ -22,204 +22,161 @@ import ballerina/mime;
 # + xAuthyKey - Unique identifier of Authy API account
 # + basicClient - HTTP client endpoint for basic api
 # + authyClient - HTTP client endpoint for authy api
-public type TwilioConnector object {
+public type TwilioConnector client object {
```

Comment:
Shall we remove this `.bal` file after moving the implementation into `twilio_endpoint.bal`.
Please refer https://github.com/wso2-ballerina/module-twitter/pull/74